packages/clerk-js/src/ui/components/SignIn/shared.ts

```
@@ -15,8 +16,9 @@ function useHandleAuthenticateWithPasskey(onSecondFactor: () => Promise<unknown>
   // @ts-expect-error -- private method for the time being
   const { setActive, __internal_navigateWithError } = useClerk();
   const supportEmail = useSupportEmail();
-  const { afterSignInUrl } = useSignInContext();
+  const { afterSignInUrl, signInUrl, navigateOnSetActive } = useSignInContext();
```

ESLint

[error] 'signInUrl' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)